apps/src/applab/commands.js

```diff
@@ -1362,6 +1362,14 @@ applabCommands.startWebRequest = function (opts) {
   req.send();
 };
 
+applabCommands.startWebRequestSync = function (opts) {
+  applabCommands.startWebRequest({...opts,
+    func: (status, contentType, responseText) => {
+      opts.func(responseText);
+    }
+  });
+};
```

Comment:
What happens if the web request fails?  It seems like it should throw - do we get that for free with `nativeIsAsync` or do we need to handle it here where we're wrapping the callback?

Reply:
The behavior of `startWebRequest` is that it calls the callback with the error.  I matched that behavior for the synchronous version (the error message is returned to the caller).

Reply:
![image](https://user-images.githubusercontent.com/413693/49046914-9dcc6880-f18a-11e8-89ee-ac13ebfc6f9a.png)